aunt-leahs-app/src/pages/VolunteerLoginPage.jsx

```diff
@@ -11,18 +11,28 @@ import AutoCompleteSelectBox from '../components/AutoCompleteSelectBox'
 import { setExistingVolunteer } from '../redux/volunteer/volunteerAction';
 import { setCurrentPage } from '../redux/page/pageAction';
 
+import { volunteerAPIBaseURL } from '../constants';
 
 function VolunteerLoginPage({ setExistingVolunteer, setCurrentPage }) {
 	const [volunteers, setVolunteers] = useState([]);
 	const [selectedVolunteer, setSelectedVolunteer] = useState(null);
 
+	const volunteerNamesEndpoint = volunteerAPIBaseURL + '/api/VolunteerNames';
+
 	useEffect(() => {
 		getVolunteers();
 	}, []);
 
 	async function getVolunteers() {
 		try {
-			const response = await fetch('http://localhost:7071/api/VolunteerNames');
+			let headers = new Headers();
```

Comment:
Why is it not possible to use the authorized fetch function here?

Reply:
This is on the volunteers side, so they wouldn't be able to log in for an auth token